Vienna Tests/models/ArticleTests.m

```diff
@@ -279,7 +279,8 @@ - (void)testExpandLinkTag
 {
     NSString *string = @"$ArticleLink$";
 
-    NSString *expectedString = [NSString stringWithFormat:@"%@/", Link];
+    //previously, we expected a '/' at the end of the link, but that was only webkit behavior and is not necessary.
```

Comment:
This is not related to webkit behaviour but related to conventions on server side (retrieve the index). Ex : https://cdivilly.wordpress.com/2014/03/11/why-trailing-slashes-on-uris-are-important/